content/es/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -0,0 +1,100 @@
+
+Para verificar esta conexión, usa la bandera `--kubelet-certificate-authority` para proporcionar al servidor API un paquete de certificados raíz para usar en la verificación del certificado de servicio del kubelet.
+
+
+Si eso no es posible, usa [túneles SSH](#ssh-tunnels) entre el servidor API y el kubelet si es necesario para evitar conectarse a través de una red no confiable o pública.
+
+Finalmente, se debe habilitar la [autenticación y/o autorización del Kubelet ](/docs/reference/access-authn-authz/kubelet-authn-authz/) para asegurar API del servicio kubelet  
+
+
+### Servdor API a nodes, pods, y services
+
+
+Las conexiones desde el servidor API a un nodo, pod o servicio por defecto son conexiones HTTP simples y, por lo tanto, no están autenticadas ni cifradas. Pueden ejecutarse sobre una conexión HTTPS segura anteponiendo `https:` al nombre del nodo, pod o servicio en la URL de la API, pero no validarán el certificado proporcionado por el punto final HTTPS ni proporcionarán credenciales de cliente. Así que, aunque la conexión estará cifrada, no ofrecerá garantías de integridad. Estas conexiones **actualmente no son seguras** para ejecutarse sobre redes no confiables o públicas.
+
+
+### Tuneles SSH
+
+
+Kubernetes soporta túneles SSH para proteger los caminos de comunicación del plano de control a los nodos. En esta configuración, el servidor API inicia un túnel SSH a cada nodo en el clúster (conectándose al servidor SSH que escucha en el puerto 22) y pasa todo el tráfico destinado a un kubelet, nodo, pod o servicio a través del túnel.
+Este túnel asegura que el tráfico no esté expuesto fuera de la red en la que se están ejecutando los nodos.
+
+
+
+
+Kubernetes supports [SSH tunnels](https://www.ssh.com/academy/ssh/tunneling) to protect the control plane to nodes communication paths. In this
+configuration, the API server initiates an SSH tunnel to each node in the cluster (connecting to
+the SSH server listening on port 22) and passes all traffic destined for a kubelet, node, pod, or
+service through the tunnel.
+This tunnel ensures that the traffic is not exposed outside of the network in which the nodes are
+running.
+
+{{< note >}}
+Los túneles SSH están actualmente en deprecados,por lo que no deberías optar por usarlos a menos que sepas lo que estás haciendo. El [servicio Konnectivity](#konnectivity-service) es un reemplazo para este canal de comunicación.
+{{< /note >}}
+
+### Servicio Konnectivity 
+
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
+
+Como reemplazo de los túneles SSH, el servicio de Konnectivity proporciona un proxy a nivel de TCP para la comunicación del plano de control con el clúster. El servicio de Konnectivity consta de dos partes: el servidor de Konnectivity en la red del plano de control y los agentes de Konnectivity en la red de los nodos. Los agentes de Konnectivity inician conexiones al servidor de Konnectivity y mantienen las conexiones de red.
```

Comment:
Previously in the above paragraphs, we have used "control plane" without translating. So, you may want to use the words in English to keep consistency.

```suggestion
Como reemplazo de los túneles SSH, el servicio de Konnectivity proporciona un proxy a nivel de TCP para la comunicación del control plane con el clúster. El servicio de Konnectivity consta de dos partes: el servidor de Konnectivity en la red del plano de control y los agentes de Konnectivity en la red de los nodos. Los agentes de Konnectivity inician conexiones al servidor de Konnectivity y mantienen las conexiones de red.
```